llama_stack/core/library_client.py

```diff
@@ -148,36 +147,23 @@ def __init__(
             config_path_or_distro_name, custom_provider_registry, provider_data
         )
         self.pool_executor = ThreadPoolExecutor(max_workers=4)
-        self.skip_logger_removal = skip_logger_removal
         self.provider_data = provider_data
 
         self.loop = asyncio.new_event_loop()
 
-    def initialize(self):
-        if in_notebook():
-            import nest_asyncio
-
-            nest_asyncio.apply()
-            if not self.skip_logger_removal:
-                self._remove_root_logger_handlers()
-
         # use a new event loop to avoid interfering with the main event loop
         loop = asyncio.new_event_loop()
         asyncio.set_event_loop(loop)
         try:
-            return loop.run_until_complete(self.async_client.initialize())
+            loop.run_until_complete(self.async_client.initialize())
         finally:
             asyncio.set_event_loop(None)
 
-    def _remove_root_logger_handlers(self):
+    def initialize(self):
```

Comment:
Would it be worth keeping this to preserve backwards compat ?

Reply:
thanks @derekhiggins for your review 👍🏽 

wdyt @ashwinb @mattf  ?

Reply:
opps, sorry I read that wrong, its being removed above and added here with `pass`

Reply:
yes I actually was looking for this review above, but could not find :) 

i recall i kept it for backward compatibility 👍🏽 

is nothing required here, shall we resolve ?